manifests/actionpolicy/rule.pp

```diff
@@ -1,13 +1,12 @@
 # Define - mcollective::actionpolicy::rule
 define mcollective::actionpolicy::rule(
-  $agent,
-  $action      = 'allow',
-  $callerid    = '*',
-  $actions     = '*',
-  $fact_filter = '*',
-  $classes     = '*'
+  String $agent,
+  String $action      = 'allow',
```

Comment:
is even something other than allow allowed here? https://github.com/puppetlabs/mcollective-actionpolicy-auth#usage